samples/SocialWeather/Startup.cs

```diff
@@ -35,7 +35,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             }
 
             app.UseSockets(o => { o.MapEndpoint<SocialWeatherEndPoint>("/weather"); });
-            app.UseStaticFiles();
+            app.UseFileServer();
```

Comment:
I thought 404 was because there was no index.html. I saw you changed that but I don't know why it requires FileServer...